src/output.rs

```diff
@@ -118,7 +118,19 @@ pub fn output_lcov(results: CovResultIter) {
             write!(writer, "FNF:{}\n", result.functions.len()).unwrap();
             write!(writer, "FNH:{}\n", result.functions.values().filter(|x| x.executed).count()).unwrap();
         }
-
+        
+        // branch coverage information
+        let mut branch_hit = 0;
+        for (&(line, number), &taken) in &result.branches {
+            write!(writer, "BRDA:{},{},{},{}\n", line, 0, number, if taken {"1"} else {"-"}).unwrap();
```

Comment:
Nit: add spaces around "1" and "-", like this: `if taken { "1" } else { "-" }` to make the code more readable.